src/modules/image/index.ts

```
@@ -45,6 +50,43 @@ export class ImageModule extends ModuleBase {
     )}`;
   }
 
+  /**
+   * Generates a random square portrait of a person (avatar). These are currently images of fake people which were generated by an AI, Stable Diffusion 3. The image URLs are served via the JSDelivr CDN and subject to their [terms of use](https://www.jsdelivr.com/terms).
+   *
+   * @param options Options for generating an AI avatar.
+   * @param options.sex The sex of the person for the avatar. Can be 'female' or 'male'. If not provided, defaults to a random selection.
+   * @param options.size The size of the image. Can be 512, 256, 128, 64, 32 or 16. If not provided, defaults to 512.
```

```suggestion
   * @param options.size The size of the image. Can be `512`, `256`, `128`, `64` or `32`. If not provided, defaults to `512`.
```

---

Or should we default to a random size?

---

I don't think we should default to a random size.